CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## Unreleased
+
+### BUG FIXES:
+
+- `[light-client]` Fix bug where a commit with only absent signatures would be
+  deemed valid instead of invalid ([#652])
+
+[#652]: https://github.com/informalsystems/tendermint-rs/pull/652
```

Comment:
Isn't it that we should reference only the issues in the changelog, not PRs? 
Also, probably it makes sense to link this PR to the issue via "closes #650"?